app/src/main/java/com/door43/translationstudio/App.java

```diff
@@ -797,6 +797,35 @@ public static void clearTargetTranslationSettings(String targetTranslationId) {
     }
 
     /**
+     * moves all settings for a target translation to new target translation (such as when target language changed)
+     * @param targetTranslationId
+     */
+    public static void moveTargetTranslationSettings(String targetTranslationId, String newTargetTranslationId) {
```

Comment:
@PhotoNomad0 can you move the migration code into core.Migration so we don't clutter the App class?